### PR TITLE
[frontend/backend] Fix Can't unlink a task from a case template (#6834)

### PR DESCRIPTION
**Pull request by SarahBocognano**
### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/6834

### CODE REVIEW

**Review by marieflorescontact — APPROVED** [review]
locally tested ok 👌

File: opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/caseTemplate-test.ts [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+      query: RELATION_ADD_QUERY,
+      variables: {
+        id: caseTemplateInternalId,
+        input: {
+          toId: taskTemplateInternalId,
+          relationship_type: 'template-task',
+        },
+      },
+    });
+    expect(queryResult.data?.caseTemplateRelationAdd).not.toBeNull();
```
Comment (labo-flg) [inline opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/caseTemplate-test.ts:154]: after here, could you call once again the `READ_QUERY` and check the tasks edges to find our task template?

**Comment by labo-flg** [comment]
nice job, thanks for the new test suite ! Case/templates were zero covered

File: opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/caseTemplate-test.ts (hunk 1 above)
Comment (SarahBocognano) [reply opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/caseTemplate-test.ts:154]: Done :)

**Pull request merged**